src/serialization/JSONACSets.jl

```diff
@@ -92,7 +92,7 @@ Inverse to [`read_json_acset`](@ref).
 """
 function write_json_acset(x::ACSet, fname::AbstractString)
   open(fname, "w") do f
-    write(f, JSON.json(generate_json_acset(x)))
+    write(f, JSON3.json(generate_json_acset(x)))
```

Comment:
this should be 
```julia
write_json_acset(x::ACSet, fname::AbstractString) = JSON3.write(fname, x)
```

because JSON3 has a method for this now.

Reply:
Done!